packages/lib/piiFreeData.ts

```diff
@@ -59,18 +59,12 @@ export function getPiiFreeBooking(booking: {
 }
 
 export function getPiiFreeCredential(credential: Partial<Credential>) {
-  return {
-    id: credential.id,
```

Comment:
There were some logs with id being unidentified because we don't strictly type check to see if all these properties are in the original credential.